src/pages/contributing/product-development-lifecycle/index.mdx

```diff
@@ -0,0 +1,145 @@
+
+</Column>
+</Row>
+
+## Discovery 
+
+The discovery phase is where research, exploration, and validation happen. This is when innovations to the system are proposed and reviewed.
+
+### Discovery criteria 
+
+For an enhancement or net new asset to warrant resourcing a discovery phase, we need to determine the requirements for an asset to be considered as an innovation. Proposals need to show that the component or pattern would be useful to many teams and unique to the system. 
+
+#### Key considerations:
+
+- Does it replicate anything in the system already, or is there truly a gap?
+- If the proposal does replicate an existing asset, is there evidence to show that the proposed solution is better?
+- Is there already an existing issue or proposal in [GitHub](https://github.com/carbon-design-system/carbon/issues) to address the gap?
+- Is there evidence that the new asset or enhancement would be useful for many teams or services?
+- What is the ratio of feasibility to impact to help prioritize (consult developers and accessibility SMEs)?
+
+#### Discovery status
+
+All assets in the discovery phase start as drafts. As the asset progresses in its completeness and validation it can graduate from `draft` to `preview candidate`. This means its ready to be reviewed and seriously considered for inclusion as a maintained asset in the system. Carbon is not the gatekeeper of the discovery phase; this phase is fully curated by the community and innovators. 
+
+#### Evaluation and next steps
+
+The Carbon team and partners will review the proposals and determine next steps. Only the Carbon team can move as asset from discovery to delivery in core Carbon or Carbon for IBM Products. 
+
+Not all proposals in the discovery phase will move on in the lifecycle, some explorations may not gain traction or may be deprioritized by other efforts. This does not mean they are not valid or shouldn’t be used; it only means it currently isn’t a priority to systematize at the core level. 
+
+It’s also important to note that Carbon does not take on ownership and maintenance of certain types of assets like full applications, third party tooling, and shared services. For example, although it is shared tooling, the Carbon team does not include the Carbon for AI chat or IBM Assist Me in its libraries. 
+
+## Delivery 
+
+In the delivery phase, the Carbon team usually collaborates with a workgroup or discovery team to begin to codify and implement their asset for preview in the Carbon library. 
+
+### Delivery criteria
+
+As a component or pattern enters the delivery phase, we begin to complete the requirements for an asset to reach "stable.” By aligning across the Carbon ecosystem on our requirements for stability—or a [definition of done](/contributing/product-development-lifecycle/#definition-of-done)—we not only share our expectations with contributors, but we can more easily create a backlog of work to prioritize, and more clearly display the status of assets to users.
```

Comment:
Wondering if you want the status "stable" here to be an inline code snippet like you did for `draft` and `preview candidate` previously on the page.